tests/conftest.py

```diff
@@ -560,6 +573,63 @@ def od_detection_mask_dataset(tiny_od_mask_data_path):
     )
 
 
+@pytest.fixture(scope="session")
+def carton_keypoint_meta():
+    return {
+        "category": "carton",
+        "skeleton": [
```

Comment:
[minor] See my comment from the notebook. When defining a COCOkeypoint_meta class (or whatever name) then can just use here rather than defining below.

Reply:
Added `carton_keypoint_meta` in `keypoint.py`